demosplan/DemosPlanCoreBundle/Logic/User/UserHandler.php

```diff
@@ -1752,30 +1752,29 @@ protected function checkMandatoryFieldsOrga($data, $currentOrga)
      *
      * @param string $organisationId Indicates the organisation whose data will be wiped
      *
-     * @return orga|bool The wiped organisation if all operations are successful, otherwise false
+     * @return orga|array The wiped organisation if all operations are successful, otherwise false
      *
      * @throws MessageBagException
      */
     public function wipeOrganisationData(string $organisationId)
     {
+        $errors = [];
         $requiredRelationsAreSolved = true;
 
         $organisation = $this->orgaHandler->getOrga($organisationId);
         if (!$organisation instanceof Orga) {
```

Comment:
The if statements after this one will not work if the $organisation is not an instance of Orga.

Reply:
done in [e6ba8d5](https://github.com/demos-europe/demosplan-core/pull/2681/commits/e6ba8d54b67ecc0082f855e0e65025252509584c)

Reply:
see https://github.com/demos-europe/demosplan-core/pull/2681#discussion_r1477914806